server/graphql/plugin/src/queries/uploaded_info.rs

```diff
@@ -95,8 +95,8 @@ impl PluginInfoNode {
                     .frontend_plugins
                     .iter()
                     .map(|r| json!({ "front_end": r.code.clone(), "types": r.types.clone() }))
-                    .collect::<HashSet<serde_json::Value>>()
-
+                    .collect::<HashSet<serde_json::Value>>(),
+            "NOTE": "Back end plugins are not currently 'stackable', i.e. only one instance of transform request requisition line plugin will be executed"
```

Comment:
This message will appear when trying to 'install' plugin via CLI (the only way to insert backend plugins atm)